app/src/routes/routesMap.js

```diff
@@ -119,7 +119,7 @@ const routesMap = {
     payload: { profileRoute: PROJECT_ASSIGNMENT_ROUTE },
   })),
 
-  [USER_PROFILE_SUB_PAGE]: `/userProfile/:profileRoute(${PROJECT_ASSIGNMENT_ROUTE}|${API_KEYS_ROUTE}|${CONFIG_EXAMPLES_ROUTE})`,
+  [USER_PROFILE_SUB_PAGE]: `organizations/:organizationSlug?/projects/:projectSlug/userProfile/:profileRoute(${PROJECT_ASSIGNMENT_ROUTE}|${API_KEYS_ROUTE}|${CONFIG_EXAMPLES_ROUTE})`,
```

Comment:
I forgot to remove it. This will need to be discussed, because now the only source of truth is URL. When we go to this URL, many components receive undefined instead of organizationSlug and projectSlug. There are many such links.